app.js

```diff
@@ -38,12 +42,25 @@ function clientLoaded (client) {
   });
 }
 
+/** Utility function for seeing if an error is fatal (crashes the program)
+ * @param {Object} err - the error in question.
+ * @return {boolean} - if the error is fatal or not
+ */
+function isFatal(err) {
+  if (err.name === 'EarlyHangup' || err.anme === 'HangupFailure' ||
```

Comment:
Since it evaluates to a boolean already, you can just use:

return err.name === 'EarlyHangup' || err.anme === 'HangupFailure' || ....;